example/src/apis/tableApiHandler.ts

```diff
@@ -0,0 +1,156 @@
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios';
+
+export class TableApiHandler {
+  private axiosInstance: AxiosInstance;
+
+  private baseUrl: string = 'https://gateway.youleap-local.io/';
```

Comment:
I wonder if it's better to use env variable / config here
Just a thought - it looks good like that too

Reply:
Great comment; I was thinking about it myself when I started working on this and came to the conclusion that because we are generating this file anyway, it will be much safer and easier to implement if it is embedded in the code.

Reply:
Also, all the files under the "example" are just there to serve as a template. they are not functional